### PR TITLE
fix(settings): #5680 dismiss tooltips on cancel in se…

### PR DESCRIPTION
**Pull request by mutdmour**
dismiss tooltips on cancel in settings panel #5680

### CODE REVIEW

File: app/scripts/views/mixins/settings-panel-mixin.js [hunk 1]
```diff
@@ -68,6 +68,8 @@ define(function (require, exports, module) {
         if (this.hideFloatingPlaceholder) {
           this.hideFloatingPlaceholder(inputEl);
         }
+        // Called to clear validation tooltips. issues/5680
+        $(inputEl).change();
```
Comment (shane-tomlinson) [inline app/scripts/views/mixins/settings-panel-mixin.js]: I suspect `$(inputEl)` needs to `$inputEls`.

Comment (shane-tomlinson) [reply app/scripts/views/mixins/settings-panel-mixin.js]: ignore me, I misread..

**Comment by shane-tomlinson** [comment]
Thank you very much for this Mutasem! This works great in testing, and the additional functional test is awesome. If you are up for taking on another bug, https://github.com/mozilla/fxa-content-server/issues/4982 is related and was discussed by our team just last week.

**Comment by mutdmour** [comment]
@shane-tomlinson Thanks for merging it :) Ya, I will look into it.

**Pull request merged**